File: C/05-Operadores Aritméticos/IncrementoDecremento.c
#include<stdio.h>

int main(void)
{
    //Temos os operadores de incremento ++ e decremento -- que acrescentam ou
    //diminuem em 1 o valor da variavel
    int i = 0;
    int j = 2;
    //O pre-incremento e decremento é feito antes
    ++i;
    --j;
    printf("%d \n",i);
    printf("%d \n",j);
    //Já o pós-incremento e decremento é feito depois
    i--;
    j++;
    printf("%d \n",i);
    printf("%d \n",j);
    return 0;
}

File: PHP/03-Operadores Aritméticos/IncrementoDecremento.php
<?php

//Temos os operadores de incremento ++ e decremento -- que acrescentam ou
//diminuem em 1 o valor da variavel

$i = 1;
$j = 2;

//O pre incremento é feito antes
echo ++$i; //Imprime 2
echo'<br/>';
echo --$j; //Imprime 1
echo'<br/>';

$x = 3;
$y = 4;

//O pós incremento é feito depois
echo $x++; //Imprime 3
echo'<br/>';
echo $y--; //Imprime 4
echo'<br/>';
//Exibindo novamente as variaveis o incremento/decremento foi feito
echo $x; //Imprime 4
echo'<br/>';
echo $y; //Imprime 3
echo'<br/>';

?>